src/test/java/unit/WorkspaceOverride.java

```diff
@@ -62,12 +65,16 @@ protected void setupOnce() throws IOException {
     TestContext.clearGlobalContextDir();
     resetContext();
 
+    workspaceCreator.login();
+
+    // grant the user's proxy group access to the bucket and dataset so that they will pass WSM's
+    // access check when adding them as referenced resources
     externalBucket = ExternalGCSBuckets.createBucket();
-    ExternalGCSBuckets.grantReadAccess(externalBucket, workspaceCreator.email);
+    ExternalGCSBuckets.grantReadAccess(externalBucket, Identity.user(workspaceCreator.email));
+    ExternalGCSBuckets.grantReadAccess(externalBucket, Identity.group(Auth.getProxyGroupEmail()));
```

Comment:
consider making the proxy group email a member on TestUsers.

Reply:
Yeah I had it that way originally. I changed it because I want to avoid as much as possible duplicating the `User` business object in the `TestUsers` class. We have to override the `login` method because domain-wide delegation doesn't use the OAuth flow. But anything we can do via CLI commands I think is better for testing.